### ISSUE TITLE
Is it the full code? It is not working

### ISSUE THREAD

**Issue by pongopal** [FupingWu90/CFDnet#1]
Hi

Is this repository has the full code? It is not working.

**Comment by FupingWu90** [comment]
Hi, the code is full,  but you have to preprocess your data  if you do not test on the CT-MR dataset as mentioned in the paper.

In fact, someone else has also contacted me , and said the code has no problem. But you might need to modify the hyperparameters of the loss weights, i.e., the 'Dis_Lamda', and the 'self.Tvalue' in 'Feature_Distribution_Distance_func'.

This CF distance could be difficult to tune, because of the difficulty of optimization by statistical gradient descent, and the computation is complex, we only use the marginal distibutions to calculate this.
It would be great if you have any idea to tackle the computation and optimization problem~

**Comment by FupingWu90** [comment]
> Hi
> 
> Is this repository has the full code? It is not working.

**Comment by FupingWu90** [comment]
> Hi
> 
> Is this repository has the full code? It is not working.

Hi, the code is full, but you have to preprocess your data if you do not test on the CT-MR dataset as mentioned in the paper.

In fact, someone else has also contacted me , and said the code has no problem. But you might need to modify the hyperparameters of the loss weights, i.e., the 'Dis_Lamda', and the 'self.Tvalue' in 'Feature_Distribution_Distance_func'.

This CF distance could be difficult to tune, because of the difficulty of optimization by statistical gradient descent, and the computation is complex, we only use the marginal distibutions to calculate this.
It would be great if you have any idea to tackle the computation and optimization problem~
